notebooks/notebooks/iclr2022/winrate-predictions.ipynb

```diff
@@ -14,16 +14,20 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #66.    plt.savefig("winrate-avg.pgf", backend="pgf")
</pre><p>I think the purple "Vic. win" line looks much more choppy because there's very few games where the victim wins (only 27 games out of 1000)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/51/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>